contribs/gmf/src/editing/editFeatureComponent.js

```diff
@@ -483,6 +491,7 @@ exports.Controller_.prototype.$onInit = function() {
     this.editableTreeCtrl.node);
   this.features = this.vectorLayer.getSource().getFeaturesCollection();
   this.tolerance = this.tolerance !== undefined ? this.tolerance : 10;
+  this.closeaftersave = this.closeaftersave !== undefined ? this.closeaftersave : false;
```

Comment:
Does it work if you remove this line and simply put the following in the constructor:
```
this.closeaftersave = false;
```

Reply:
it works, done